plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureStorageSettings.java

```diff
@@ -189,10 +194,6 @@ public String getName() {
         return name;
     }
 
-    public String getKey() {
```

Comment:
I removed this API here as well as adding the `key` to the `toString` output, just like it was done in `master` when SAS token support was added.